OpenVnmrJ/ovjTools
# Updates for VM build scripts.

- added script install_test.sh for gui-less install
- updates to install/test within VM functionality
- move install_test.sh to OpenVnmrJ repo.
- fixes for vm vjqa
- add an option to pass flags to install_test.sh (ie -F '-c inova')
- treat fedora like centos
- added some missing rpm packages for centos7
- moved logging stuff to loglib.sh shell-function library
- default log file name is now build_vm...log
- take ubuntu18 and macos10.10 out of 'all' target until they work
- catch a bunch of error conditions
- retrieve VJQA test result as log file named vjqa_....txt
- make sure vagrant setup scripts fail on error with set -e
- default the debian installations to use bash as /bin/sh (not dash!)
- started (an unfinished) 'package' option to build .rpm/.deb
- log runtimes of various build stages
- checkout less git depth for shallow builds
- use 4 cpus on VMs instead of 2 for faster build.
- make Vagrantfile installs fail if packages dont install
